packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/connectors.module.js

```diff
@@ -0,0 +1,28 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ui-kit';
+
+import component from './connectors.component';
+import routing from './connectors.routing';
+
+import restartButton from '../../components/restart-button';
```

Comment:
**suggestion**: Creating custom components can be avoided.

Reply:
This component is also used in another page (in this PR: https://github.com/ovh/manager/pull/6582)